### ISSUE TITLE
Strip disabled artifacts from bundle

### ISSUE THREAD

**Issue by x1B** [LaxarJS/laxar-tooling#21]
Items (widgets, layouts, compositions) that are _disabled_ (by setting `enabled` to false) should be stripped from the bundle along with their dependencies.

**Comment by jpommerening** [comment]
@x1B: Do we need to be able to handle `"enabled": "${features.includeWhatever.enabled}"` in compositions as well?

**Comment by x1B** [comment]
I'd say *nice to have*. We just need to add clear documentation to the *manual on compositions* regarding when which interpolations are supported.

**Comment by jpommerening** [comment]
Ok. If we want to be able to validate/interpolate during runtime in the future, we'll have to keep all the "possibly enabled" artifacts in the bundle anyway.

**Comment by x1B** [comment]
agreed

**Comment by jpommerening** [comment]
Fixed on `master` (`v0.7.0`)

## BREAKING CHANGE

Artifacts referenced from pages and marked with `"enabled": false` will now be omitted from the generated artifact listing (unless they are referenced somewhere else). If, for some reason, you need those artifacts present, you need to name them as entry-points to the artifact collector. Artifact dependencies like controls will then be resolved as if the artifact had been listed in a page.

```js
// using the laxar-tooling API
import { artifactCollector } from 'laxar-tooling';

const collector = artifactCollector.create( { resolve, readJson } );
collector.collectArtifacts( [ {
   flows: [ 'main' ],
   widgets: [ 'my-disabled-widget' ],
   layouts: [ 'my-disabled-layout' ]
} ] ).then( artifacts => { /* ... */ } );

// using laxar-loader:
import artifacts from 'laxar-loader/artifacts?flow=main&widgets[]=my-disabled-widget&layouts[]=my-disabled-layout';
```
